src/main/java/com/xebialabs/deployit/ci/DeployitNotifier.java

```diff
@@ -253,6 +257,7 @@ public int getConnectionPoolSize() {
             return connectionPoolSize;
         }
 
+        @RequirePOST
```

Comment:
While not harmful isn't necessary. Leave it in or remove.

Reply:
Removed the unnecessary annotation.